### PR TITLE
refactor: remove navbar breadcrumbs

### PR DESCRIPTION
**Pull request by OmranAbazid**
## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

related to #23830

### CODE REVIEW

File: optimize/client/e2e/cloud-tests/smokeTest.js
Comment (misiekhardcore) [inline optimize/client/e2e/cloud-tests/smokeTest.js:1]: There is an unused import in this file

File: optimize/client/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -94,7 +93,6 @@ function createAppProps(location: {pathname: string}): C3NavigationProps['app']
     name: t('appName').toString(),
     ariaLabel: t('appFullName').toString(),
     routeProps: {
-      as: Link,
       className: 'cds--header__name',
```
Comment (misiekhardcore) [inline optimize/client/src/components/Header/Header.tsx:96]: I think this `className` is not needed anymore

**Review by misiekhardcore — APPROVED** [review]
Pre-approving
Code-wise it looks good and I'm glad that we finally removed the breadcrumbs , but now we miss the functionality they provided

**Comment by OmranAbazid** [comment]
> I see you just removed the breadcrumbs entirely. Is there any plan to add it in a different form? using the `C3Page` for instance? Right now we lost the indication of where we are completely. Especially opening a dashboard/report inside of a collection

Yes, I will be working on adding the C3 page

**Pull request merged**